<launch>

	<node name="theatre_uav_trajectory_planner" pkg="theatre_uav_trajectory_planner" type="trajectory_planner2" output="screen">
		<param name="rate" value="10.0"/>
		<param name="duration" value="20.0"/>
		<param name="x_square" value="0.6"/>
		<param name="y_square" value="0.4"/>
		<param name="altittude_square" value="0.8"/>
		<param name="altittude" value="1.3"/>
		<param name="time_of_state" value="30.0"/>  <!-- max 35 -->
		<param name="disarming_z" value="2.42"/>
	</node>


</launch>
